cv32e40s/tests/cfg/pma_test_cfg_1.yaml

```diff
@@ -1,10 +1,15 @@
 name: pma_test_cfg_1
 description: PMA configuration for the PMA_TEST_CFG_1 test case
-compile_flags:
+compile_flags: >
    +define+PMA_TEST_CFG_1
    +define+ZBA_ZBB_ZBC_ZBS
-plusargs:
+plusargs: >
```

Comment:
Was this broken and non-functional before adding the `>`?

Reply:
Added for consistency